protobuf/video/sfu/models/models.proto

```diff
@@ -231,4 +238,10 @@ message CallGrants {
   bool can_publish_audio = 1;
   bool can_publish_video = 2;
   bool can_screenshare = 3;
+}
+
+enum GoAwayReason {
+  GO_AWAY_REASON_UNSPECIFIED = 0;
+  GO_AWAY_REASON_SFU_TERMINATING = 1;
```

Comment:
`GO_AWAY_REASON_SERVER_SHUTTING_DOWN`

Reply:
also fine keeping this name if too much work to rename

Reply:
I'll fix it.